src/GHC/Check/Util.hs

```diff
@@ -4,13 +4,15 @@
 {-# LANGUAGE CPP, TemplateHaskell #-}
 module GHC.Check.Util (MyVersion(..), liftTyped, gcatchSafe) where
 
-import           Control.Exception.Safe
+import           Control.Exception.Safe as Safe
 import           Control.Monad.IO.Class (MonadIO(liftIO))
 import           Data.Version ( Version, parseVersion )
-import           GHC (Ghc, gcatch)
+import           GHC (Ghc)
+import qualified GHC
```

Comment:
I'm not sure if I should add CPP pragmas to avoid these "Redundant imports" warnings

Reply:
i usually do it to clean up warnings but to make explicit what are we picking for each ghc version

Reply:
Makes sense